datastream-kafka-connector/src/main/java/com/linkedin/datastream/connectors/kafka/KafkaBasedConnectorConfig.java

```diff
@@ -32,6 +32,7 @@
   public static final String CONFIG_PAUSE_PARTITION_ON_ERROR = "pausePartitionOnError";
   public static final String CONFIG_PAUSE_ERROR_PARTITION_DURATION_MILLIS = "pauseErrorPartitionDurationMs";
   public static final String ENABLE_ADDITIONAL_METRICS = "enableAdditionalMetrics";
+  public static final String INCLUDE_DATASTREAM_NAME_IN_CONSUMER_CLIENT_ID = "includeDatastreamNameInConsumerClientId";
```

Comment:
Do we need to make this a configurable? Why not add it without the config?

Reply:
Want to make this configurable for two reasons:

1. Kafka quotas are currently enforced based on client.id. Kafka is in the process of migrating away from this model, but I don't want to break the client.ids until we are sure that the migration is complete and successful. So was thinking we can enable this on a per cluster basis.
2. I don't expect us to hit metrics limitations in BMM, but want to make sure we have a knob to turn this off in case we start having problems with metrics.

Once we roll this out everywhere and all looks good, I can make a new PR to remove the config. How does that sound?